documentation/src/main/asciidoc/topics/con_performance_metric_considerations.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: CONCEPT
+[id='performance-metric-considerations_{context}']
+= Performance metric considerations
+
+{brandname} includes so many configurable combinations that determining a single formula for performance metrics that covers all use cases is not possible.  
+
+Additionally, consider the following inter-related factors that apply to {brandname}:
+
+* Available CPU and memory resources in cloud environments
+* Caches used in parallel
+* Get, put, query balancing 
+* Peak load and throughput limitations
+* Querying limitations with data set
+* Number of entries per cache
+* Size of cache entries
+
+Given the number of different combinations and unknown external factors, providing a performance calculation that meets all {brandname} use cases is not possible, because you cannot compare one performance test to another test if any of the previously listed factors are different. 
+
+The purpose of this document is to provide details about use cases and architectures that can help you determine requirements for your {brandname} deployment.
```

Comment:
I would move this sentence to the top. Maybe the opening sentence.

Reply:
Moved to start of module.